### PR TITLE
docs(HOWTO): Translated alot of Text in Hindi and added few links

### PR DESCRIPTION
**Pull request by guptamanu409**
##My first contributions under Hacktober Fest 2022
Amazing project really glad to help you guys

## What does this PR do?
-Translated Text to Hindi
-Added website link 

Add resource(s) | Remove resource(s) | Add info | Improve repo

### Why is this valuable (or not)?
-I am from India where Hindi is our Mother tongue. So i made changes to improve it further.
### How do we know it's really free?
-I want to make the World a better place, can't charge for that.
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: docs/HOWTO-hi.md [hunk 1]
```diff
@@ -10,18 +10,18 @@
 
 हम नए योगदानकर्ताओं का स्वागत करते हैं; यहां तक ​​कि उन लोगों के लिए जो GitHub पर अपना पहला Pull Request (PR) करते हैं। यदि आप उनमें से एक हैं, तो यहां कुछ संसाधन हैं जो मदद कर सकते हैं:
 
-* [About Pull Requests](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests) *(in english)*
-* [Creating a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) *(in english)*
-* [GitHub Hello World](https://docs.github.com/en/get-started/quickstart/hello-world) *(in english)*
-* [YouTube - GitHub Tutorial For Beginners](https://www.youtube.com/watch?v=0fKg7e37bQE) *(in english)*
-* [YouTube - How To Fork A GitHub Repo and Submit A Pull Request](https://www.youtube.com/watch?v=G1I3HF4YWEw) *(in english)*
-* [YouTube - Markdown Crash Course](https://www.youtube.com/watch?v=HUBNt18RFbo) *(in english)*
+* [पुल अनुरोध के बारे में](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests) *(Pull Request)*
```
Comment (eshellman) [inline docs/HOWTO-hi.md:13]: prefer to leave the annotation as *(in english)* The rest look good!

Comment (guptamanu409) [reply docs/HOWTO-hi.md:13]: > prefer to leave the annotation as _(in english)_ The rest look good!

I'm soo sorry Sir that i didn't saw your message..... I'm correcting it right now sir.

Comment (davorpa) [reply docs/HOWTO-hi.md]: Solved at 4ee30c7d74ea6b4f5bdf248932fe56384418f12f

**Review by davorpa — APPROVED** [review]

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**